packer/ubuntu-build.json

```diff
@@ -11,7 +11,7 @@
     "audio": "pulse",
     "output_dir": "{{pwd}}/artifacts_ubuntu",
     "mirror_url": "https://mirror.cs.jmu.edu/pub/ubuntu-iso/focal",
-    "iso_file": "ubuntu-20.04.3-desktop-amd64.iso",
+    "iso_file": "ubuntu-20.04.4-desktop-amd64.iso",
```

Comment:
Was wondering why our automated workflow didn't catch this...

![image](https://user-images.githubusercontent.com/850893/160261919-e0f4121e-89f6-4f7a-99b8-04f33900e53a.png)

😢